python/hyperon/exts/snet_io/snet_gate.py

```diff
@@ -137,9 +139,105 @@ def get_service_messages(self):
     def get_operation_atom(self):
         return OperationAtom(self.service_details[1], self)
 
+    def __pretty_print__(self, input_str):
```

Comment:
pretty-print was supposed for adding to stdlib, but maybe it can remain here for now....

Reply:
Sorry, I wasn't been aware of this. Where should I place it?